Google.Api.Gax.Grpc/Rest/RestServiceCollection.cs

```diff
@@ -37,7 +37,8 @@ internal static RestServiceCollection Create(ApiMetadata metadata)
             var typeRegistry = TypeRegistry.FromFiles(fileDescriptors.ToArray());
             var parser = new JsonParser(JsonParser.Settings.Default.WithIgnoreUnknownFields(true).WithTypeRegistry(typeRegistry));
             var methodsByName = services.SelectMany(service => service.Methods)
-                .Where(x => true) // TODO: filter out streaming methods.
+                // We don't yet support streaming methods.
+                .Where(x => !x.IsClientStreaming && !x.IsServerStreaming)
```

Comment:
Both of these will be true for bidi I think, else we need a specific check.

Reply:
Yup, I'm pretty sure they're both true for bidi.